parts/inc/SvPV

```diff
@@ -424,8 +426,14 @@ SvPV_renew(sv, nlen, insv)
                 SvCUR_set(sv, slen);
                 mXPUSHi(SvLEN(sv));
 
+void
+SvPVCLEAR(sv)
+        SV *sv
+        CODE:
+                SvPVCLEAR(sv);
```

Comment:
this reads weird to use the same macro name than the C function, no?
should we not prefer a different name for the macro?

Reply:
I think this is correct and IIRC preferred naming (I think it is described in some xs doc).

Anyway, XS language allows you to declare XS function without body and say let XS compiler to automatically generate code which calls C function with the same name.

So if this style is going to used, I would suggest to declare this SvPVCLEAR XS function in a way that XS compiler automatically generates its CODE section.

Reply:
Ok, I guess then let it as it is now.